mne/io/_pick.py

```diff
@@ -0,0 +1,27 @@
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD-3-Clause
+
+
+from .._fiff import _io_dep_getattr
+from .._fiff.pick import (
+    _picks_to_idx,
+    _DATA_CH_TYPES_ORDER_DEFAULT,
+    _DATA_CH_TYPES_SPLIT,
+)
+
+__all__ = [
+    # mne-bids, autoreject, mne-connectivity, mne-realtime, mne-nirs, mne-realtime
+    "_picks_to_idx",
+    # mne-qt-browser
+    "_DATA_CH_TYPES_ORDER_DEFAULT",
+    "_DATA_CH_TYPES_SPLIT",
+]
```

Comment:
@drammock other than `mne.io.constants.FIFF` (which is pretty painless to keep) this is the list of what we have to keep around to make submodules happy. Do you think we should make some variant of `_picks_to_idx` public?

Reply:
yes I do.  I notice it was used in MNE-BIDS and it's a pretty common thing to want to do (I believe @Aaronearlerichardson mentioned something about it in #11899 today in fact)